src/common/mmo.hpp

```diff
@@ -83,7 +83,7 @@ typedef uint32 t_itemid;
 #define MAX_BANK_ZENY SINT32_MAX ///Max zeny in Bank
 #define MAX_FAME 1000000000 ///Max fame points
 #define MAX_CART 100 ///Maximum item in cart
-#define MAX_SKILL 1454 ///Maximum skill can be hold by Player, Homunculus, & Mercenary (skill list) AND skill_db limit
```

Comment:
#define MAX_SKILL 1455 Gets conflict with athena web-service of vstrumpf
i reproduce and skill hotkeys not saving.

Reply:
This is not related to this PR and I think you did something wrong.